src/root.component.tsx

```diff
@@ -3,11 +3,21 @@ import { Route } from "react-router";
 import { BrowserRouter } from "react-router-dom";
 import openmrsRootDecorator from "@openmrs/react-root-decorator";
 import PatientChartSummary from "./summary/patient-chart-summary.component";
+import PatientChartLevelTwo from "./summary/patient-chart-level-two.component";
 
 function Root(props) {
   return (
-    <BrowserRouter>
-      <Route to="/patient/:patientUuid/chart" component={PatientChartSummary} />
+    <BrowserRouter basename={window["getOpenmrsSpaBase"]()}>
```

Comment:
Is this to make typescript/eslint happy? You can also define the global variables, but I think what you've done here is fine (and probably easier)